README.md

```diff
@@ -1,17 +1,126 @@
-# reports-js-client
+# clever-goals.js
 
-Javascript client library for the public Goals API
+Javascript client library for the Clever Goals API.
+
+## Usage
+
+### Quickstart
+
+1. Download [dist/clever-goals.js](./dist/clever-goals.js) and add it to your project.
+
+2. Load the JS library in your HTML:
+
+    ```html
+    <script src="clever-goals.js" type="text/javascript"></script>
```

Comment:
@samfishman what do you think about hosting the JS for our apps - so this line would look like:
``` <script src="https://assets.clever.com/clever-goals.js" type="text/javascript"></script>
```
That's roughly how Google Analytics does it, right?